helevents/models.py

```diff
@@ -228,11 +229,31 @@ class User(AbstractUser, UserModelPermissionMixin, SerializableMixin):
         {"name": "last_name"},
         {"name": "email"},
         {"name": "signup_created_by"},
+        {"name": "events_event_created_by"},
+        {"name": "publisher_organizations"},
     )
 
     def __str__(self):
         return " - ".join([self.get_display_name(), self.email])
 
+    @property
+    def publisher_organizations(self):
+        """
+        Organizations where user is part of as admin or regular user.
+        This is used in the gdpr api user data serialization.
+        """
+
+        publishers = Organization.objects.filter(
+            Q(admin_users=self) | Q(regular_users=self)
+        )
+
+        return [
+            org.serialize()
+            for org in SerializablePublisher.objects.filter(
+                id__in=publishers.values_list("id", flat=True)
+            )
+        ]
```

Comment:
Would this work as well? 🤔 
```suggestion
        return [
            org.serialize()
            for org in SerializablePublisher.objects.filter(
                Q(admin_users=self) | Q(regular_users=self)
            )
        ]
```

Reply:
yes it would 🙃  
There might have been readability / self documenting reasoning but I don't see it anymore 😄